tests/support_server/wait.pm

```diff
@@ -21,25 +20,9 @@ use lockapi;
 use mmapi;
 
 sub run {
-    $testapi::username = "root";
-    $testapi::password = "linux";
 
+    my $self = shift;
 
-    assert_screen( "autoyast-system-login-console", 300 );
-    
-    type_string "$username\n";
-    sleep 5;
-#        assert_screen "password-prompt", 10;
-    type_password;
-    type_string "\n";
-    sleep 5;
-    
-    
-    script_output("
-        curl -f -v " . autoinst_url . "/data/supportserver/proxy.conf | sed -e 's|#AUTOINST_URL#|" . autoinst_url . "|g' >/etc/apache2/vhosts.d/proxy.conf
-        systemctl restart apache2.service
-    ");
-    
```

Comment:
I would add at least *wait_idle* for the peace of the mind, and/or comment that this really should be loaded after **support_server/setup.pm**. 
The name of the test **wait.pm** is not very descriptive for other IMO, what about **wait_for_children.pm**?